scene/main/window.cpp

```diff
@@ -1251,6 +1251,13 @@ Rect2i Window::get_parent_rect() const {
 	}
 }
 
+void Window::set_clamp_to_embedder(bool p_enable) {
+	clamp_to_embedder = p_enable;
+}
+bool Window::is_clamped_to_embedder() const {
```

Comment:
Remember to always add a separation line between method definitions, that's our style guide.